java/arcs/android/devtools/devtools_message.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package arcs.android.devtools;
+
+import "java/arcs/android/storage/proxy_message.proto";
+import "google/protobuf/wrappers.proto";
```

Comment:
You can remove this now

Reply:
I don't think I can? The message is still a wrapper around the ProxyMessageProto, it's the first field.